recipe/run_test.py

```diff
@@ -15,6 +15,7 @@
 assert ver_string == ruamel_yaml.__version__
 
 
+"""
```

Comment:
Well, looks like there was an unannounced breaking change in https://github.com/conda/conda/commit/86cb184548215b034c2e447baa6dd366b1b1a9e1#diff-aa5c1e6732e4802903e91fe909341fd0L380.
Fixing the test is as simple as
```diff
-check_call('conda config --add channels _test_channel_'.split(), env=env)
+check_call('conda config --add channels _test_channel_'.split() + ['--file={}'.format(condarc_path)], env=env)
```

It's still ugly to have these tests here, though, of course.
We should revisit the downstream testing thing. But I don't know how that handles chicken-egg-problems, e.g., testing `ruamel{.,_}yaml` built for `python=3.8` when `conda` isn't yet available for that...